### ISSUE TITLE
Unable to disable Proxy Redirect URI on non-local site

### ISSUE THREAD

**Issue by pl-mnm** [verbb/formie#1164]
### Describe the bug

I'm integrating Formie with Google Forms. I have disabled the Proxy Redirect URI setting locally, and after pushing the project config to staging this option remains enabled.

On my local environment, if I define the Proxy Redirect setting via an environment variable, it will get saved in the project config as a boolean instead of the `$FORMIE_INTEGRATION_PROXY_REDIRECT` env var.

I'm a little bit confused as to what the intended behaviour is.

Thanks in advance.

### Steps to reproduce

1. Define `$FORMIE_INTEGRATION_PROXY_REDIRECT` with `false` in `.env`
2. Create new Google Sheets integration, set Proxy Redirect URI to  `$FORMIE_INTEGRATION_PROXY_REDIRECT`
3. Project config will save it as false instead of env var.
4. Push to staging, Proxy Redirect URI remain `true`

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

3.7.59

### Plugin version

1.6.17

### Multi-site?

No

**Comment by engram-design** [comment]
Looks like the issue is saving the .env variable for the proxy redirect URI field. Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by pl-mnm** [comment]
The fix works, thanks

**Comment by engram-design** [comment]
Fixed in [1.6.18](https://github.com/verbb/formie/releases/tag/1.6.18)
